<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="7">
          <widget class="QSlider" name="cannyLowTHSlider">
           <property name="maximum">
            <number>250</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>V High</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSlider" name="kernelSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>15</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QSlider" name="valueHighTHSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QSlider" name="cannyHighTHSlider">
           <property name="maximum">
            <number>250</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>V Low</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QSlider" name="valueLowTHSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Kernel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>C H</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>C L</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="spinBox"/>
         </item>
         <item row="1" column="4">
          <widget class="QLCDNumber" name="lcdNumber">
           <property name="styleSheet">
            <string notr="true">background-color: black;
color: rgb(0, 239, 17);</string>
           </property>
           <property name="digitCount">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLCDNumber" name="lcdNumber_2">
           <property name="styleSheet">
            <string notr="true">background-color: black;
color: rgb(0, 239, 17);</string>
           </property>
           <property name="digitCount">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLCDNumber" name="lcdNumber_3">
           <property name="styleSheet">
            <string notr="true">background-color: black;
color: rgb(0, 239, 17);</string>
           </property>
           <property name="digitCount">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLCDNumber" name="lcdNumber_4">
           <property name="styleSheet">
            <string notr="true">background-color: black;
color: rgb(0, 239, 17);</string>
           </property>
           <property name="digitCount">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="1">
          <widget class="QSlider" name="thetaSlider">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>30</number>
           </property>
           <property name="pageStep">
            <number>180</number>
           </property>
           <property name="value">
            <number>180</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Theta</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QSlider" name="maxGapSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSlider" name="THSlider">
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>MinLen</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSlider" name="minLenSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QSlider" name="rhoSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>RHO</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>MaxGap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLCDNumber" name="lcdNumber_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 239, 17);
background-color: black;</string>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="lcdNumber_6">
           <property name="styleSheet">
            <string notr="true">background-color: black;
color: rgb(0, 239, 17);</string>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>3</number>
           </property>
           <property name="mode">
            <enum>QLCDNumber::Dec</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLCDNumber" name="lcdNumber_7">
           <property name="styleSheet">
            <string notr="true">background-color: black;
color: rgb(0, 239, 17);</string>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLCDNumber" name="lcdNumber_8">
           <property name="styleSheet">
            <string notr="true">background-color: black;
color: rgb(0, 239, 17);</string>
           </property>
           <property name="digitCount">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLCDNumber" name="lcdNumber_9">
           <property name="styleSheet">
            <string notr="true">background-color: black;
color: rgb(0, 239, 17);</string>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
